drillx/kernels/orev2.cu

```diff
@@ -66,7 +71,8 @@ __global__ void kernel_start_drill(
     uint8_t *d_challenge,
     uint64_t stride,
     uint64_t round,
-    uint32_t batch_size)
+    uint32_t batch_size,
+    uint64_t *output)
 {
     uint64_t iters = 0;
     uint64_t nonce = threadIdx.x + (blockIdx.x * blockDim.x) + (round * stride * batch_size);
```

Comment:
you can remove local_best_difficulty and local_best_nonce, not used anymore

Reply:
ok, let me checkl